src/ResourceManager/Network/Commands.Network/VirtualNetworkGatewayConnection/NewAzureVirtualNetworkGatewayConnectionCommand.cs

```diff
@@ -134,7 +134,7 @@ public class NewAzureVirtualNetworkGatewayConnectionCommand : VirtualNetworkGate
         [Parameter(
             Mandatory = false,
             HelpMessage = "Use policy-based traffic selectors for a S2S connection")]
-        public SwitchParameter UsePolicyBasedTrafficSelectors { get; set; }
+        public string UsePolicyBasedTrafficSelectors { get; set; }
```

Comment:
@henry416 why was this change made? You should **never** try to parse a string to a bool.

It looks like `vnetGatewayConnection.UsePolicyBasedTrafficSelectors` is false by default, so if a user wants to make it true, then they just use the SwitchParameter that is currently offered. There should be no reason to change this functionality and introduce a breaking change.